smoke-test.sh

```diff
@@ -349,6 +355,14 @@ fi
 export NUGET_PACKAGES="$restoredPackagesDir"
 SOURCE_BUILT_PKGS_PATH="$SCRIPT_ROOT/bin/obj/x64/$configuration/blob-feed/packages/"
 
+# XXX crummel XXX
+# This is a temporary workaround to disable tests that will fail until ASP.NET packages are updated.
+# See https://github.com/dotnet/source-build/issues/635 for more details.
+echo "WARNING: Ignoring parameters to disable known-failing tests." | tee -a "$logFile"
```

Comment:
(Nit) `Ignoring parameters` sounds kinda alarmingly general, I'd just list `Setting excludeWebTests and excludeOnlineTests to disable known-failing tests.` even though it's a little bit of extra maintenance in theory.

Reply:
Changed as suggested.